packages/components/src/_styles/index.js

```diff
@@ -0,0 +1,3 @@
+export * from './reduceMotion';
```

Comment:
I ended up doing something similar with the files I created under `/utils

Reply:
I don't mind either way. I've moved it all into `utils` for now.